homeassistant/components/openweathermap/const.py

```diff
@@ -146,7 +147,7 @@
     "rainy": [300, 301, 302, 310, 311, 312, 313, 500, 501, 520, 521],
     "snowy": [600, 601, 602, 611, 612, 620, 621, 622],
     "snowy-rainy": [511, 615, 616],
-    "sunny": [800],
+    "sunny/clear-night": [WEATHER_CODE_SUNNY_OR_CLEAR_NIGHT],
```

Comment:
`sunny/clear-night` is not a valid Home Assistant condition.

Reply:
My thoughts on that are here https://github.com/home-assistant/core/pull/42982#discussion_r519493539, but if necessary I can revert my change and use a HA condition.

Reply:
We need to stick to the valid values.

Reply:
Fair enough, will change it asap.